pkg/pool-weighted/contracts/smart/ManagedPool.sol

```diff
@@ -370,8 +388,13 @@ contract ManagedPool is BaseWeightedPool, AumProtocolFeeCache, ReentrancyGuard {
     }
 
     /**
-     * @dev Schedule a gradual weight change, from the current weights to the given endWeights,
-     * over startTime to endTime.
+     * @notice Schedule a gradual weight change.
+     * @dev The weights will change from their current values to the given endWeights, over startTime to endTime.
```

Comment:
Maybe add a mention here on why we must start from the current weights for completeness?

Reply:
Yeah, I think that would be sensible as it would allow you to perform the interpolation offchain while only having to do a single call. I see you've done this already in #1324 .